internal/resources/resource_volume.go

```diff
@@ -117,6 +118,11 @@ func volumeSchema() map[string]*schema.Schema {
 			Optional:    true,
 			Description: "volume labels as (name, value) pairs",
 		},
+		vWWN: {
+			Type:        schema.TypeString,
+			Optional:    true,
```

Comment:
If the volume WWN comes from storage (as the volume serial number), this should be declared like vState and vStatus, with:
   Computed:    true,
instead of:
   Optional:    true,

Reply:
Fixed as suggested